src/wip/streaming-dca.rain

```diff
@@ -0,0 +1,188 @@
+# Strategy: Streaming DCA
+
+networks:
+  polygon-red: 
+    rpc: https://rpc.ankr.com/polygon 
+    chain-id: 137 
+    network-id: 137 
+    currency: MATIC
+
+subgraphs:
+  polygon-red: https://api.thegraph.com/subgraphs/name/h20liquidity/polygon-0xc95a5f8e
+  
+orderbooks:
+  polygon-red:
+    address: 0xc95A5f8eFe14d7a20BD2E5BAFEC4E71f8Ce0B9A6
+    network: polygon-red
+    subgraph: polygon-red
+
+deployers:
+  polygon-red:
+    address: 0xB3aC858bEAf7814892d3946A8C109A7D701DF8E7
+    network: polygon-red
+
+tokens:
+  polygon-red:
+    network: polygon-red
+    address: 0x222789334D44bB5b2364939477E15A6c981Ca165
+  polygon-blue:
+    network: polygon-red
+    address: 0x6d3AbB80c3CBAe0f60ba274F36137298D8571Fbe
+
+scenarios:
+     streaming-dca:
+        network: polygon-red
+        deployer: polygon-red
+        orderbook: polygon-red
+        bindings:
+          # Ask for now, registry in future.
+          uniswap-words: 0x2382e861cF4F47578aC29B50944b3b445577aF74
+          orderbook-subparser: 0x8f037f2a3fF2dee510486D9C63A47A245991a4C1
+        scenarios:
+          red:
+            bindings:
+              # Input and Output token addresses from perspective of order
+              input-token-address: 0x222789334D44bB5b2364939477E15A6c981Ca165
+              output-token-address: 0x6d3AbB80c3CBAe0f60ba274F36137298D8571Fbe
+    
+              # Number of output tokens offered per second.
+              per-second-rate: 0.1
+    
+              # Unix timestamp for strating of the strategy.
+              #Ideally set it 15 minutes ahead of the whatever the timestamp is while deploying
+              strat-strat-time: 1718738185
```

Comment:
start this at a very high number like 9999999999 so that it never accidentally drains, worst case is that it just accidentally never starts